index.bs

```diff
@@ -444,13 +444,16 @@ When this method is invoked, the user agent MUST execute the following algorithm
 8. Initialize |issuedRequests| and |currentlyAvailableAuthenticators| to empty lists.
 
 9. For each authenticator currently available on this platform, add the authenticator to |currentlyAvailableAuthenticators|
-    unless the {{ScopedCredentialOptions/attachment}} member of {{options}} is <a>present</a>. In that case, let |attachment|
-    be {{ScopedCredentialOptions/attachment}}, and add the authenticator to |currentlyAvailableAuthenticators| if its attachment
-    modality matches |attachment|.
+    unless the {{ScopedCredentialOptions/attachment}} member of {{options}} is present or the 
```

Comment:
Try not to use this "do x unless y. If y, z." form. Instead, it's clearer to write out "If y, z. Otherwise, x".

You'll probably need to indent the content of the "[for each](https://infra.spec.whatwg.org/#list-iterate)" and maybe name |authenticator|, but that's not really a problem.